src/interface/interface.ts

```diff
@@ -326,6 +326,7 @@ export interface TxHistoryQueryParams {
 export interface User {
   user_address: string;
   balance: Uint8Array;
+  credits: string;
```

Comment:
2 points need discuss:
1. We are changing from balance to credits, so should we remove balance? We need have reason why we should/should not remove balance attribute.
2. credits and balance are both U256 in b/e. Why credits is string and balance is Uint8Array? Which one is better?